package.json

```diff
@@ -1,29 +1,18 @@
 {
```

Comment:
I can't comment on a file in Github (boo!), but I moved all development dependencies to the correct place.